pkg/block/fetcher.go

```diff
@@ -484,9 +495,26 @@ func (f *DeduplicateFilter) filterForResolution(root *Node, metaSlice []*metadat
 }
 
 // DuplicateIDs returns slice of block ids
-// that are filtered out by DeduplicateFilter.
-func (f *DeduplicateFilter) DuplicateIDs() []ulid.ULID {
-	return f.duplicateIDs
+// that are filtered out by DeduplicateFilter and not marked to be deleted.
+func (f *DeduplicateFilter) DuplicateIDs() ([]ulid.ULID, error) {
+	toFilterIds := []ulid.ULID{}
```

Comment:
```suggestion
	toFilterIDs := []ulid.ULID{}
```
as per our style guide.

Reply:
yes, made this change